src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -80,14 +82,29 @@ class RocketSubsystem(implicit p: Parameters) extends BaseSubsystem
   // add Mask ROM devices
   val maskROMs = p(PeripheryMaskROMKey).map { MaskROM.attach(_, cbus) }
 
+  val hartPrefixNode = if (p(HartPrefixKey)) {
```

Comment:
If possible, I would put this in the trait `HasRocketTiles`, which actually currently gets reused across multiple `BaseSubsystem` subclasses, unlike `RocketSubsystem` which is only used here and in `soc-testsocket-sifive`

Reply:
I would do that, but then this PR would break those other subclasses. Since the hartReg() part of this is needed to unblock the pipeline, I think I will do that refactor as a follow-up PR.